extensions/harvest/src/services/harvest.ts

```diff
@@ -92,28 +94,25 @@ export function useActiveClients() {
 
 async function fetchProjects() {
   let project_assignments: HarvestProjectAssignment[] = [];
-  let page = 1;
-  // eslint-disable-next-line no-constant-condition
-  while (true) {
+  let pageParams = {};
+
+  while (pageParams) {
```

Comment:
This should remain while (true)
```suggestion
  while (true) {
```